<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Kunal-Lenovo" name="Org.qsp.testngdemo.DataExcelDemo" tests="1" failures="0" timestamp="30 Jul 2020 06:36:03 GMT" time="0.000" errors="1">
  <testcase name="m1" time="0.000" classname="Org.qsp.testngdemo.DataExcelDemo">
    <error type="java.lang.RuntimeException" message="Can&#039;t invoke method public java.lang.String[][] Org.qsp.testngdemo.ReadData.getData() throws java.lang.Exception on this instance of class Org.qsp.testngdemo.DataExcelDemo due to class mismatch">
      <![CDATA[java.lang.RuntimeException: Can't invoke method public java.lang.String[][] Org.qsp.testngdemo.ReadData.getData() throws java.lang.Exception on this instance of class Org.qsp.testngdemo.DataExcelDemo due to class mismatch
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:73)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:136)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:441)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1303)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1019)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1108)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1200)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1125)
	at org.testng.TestNG.run(TestNG.java:1033)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- m1 -->
</testsuite> <!-- Org.qsp.testngdemo.DataExcelDemo -->
